src/main/java/Race.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+
+public class Race {
+    int timeRace = 24;
```

Comment:
Это можно сделать константой следующим образом:
```suggestion
    private static final int timeRace = 24;
```